src/text-fragment-utils.js

```diff
@@ -835,20 +835,41 @@ if (typeof goog !== 'undefined') {
  * text-fragments-polyfill-target-text
  *
  */
-export const applyTargetTextStyle =
-    () => {
-      const styles = document.getElementsByTagName('style');
-      if (!styles) return;
-
-      for (const style of styles) {
-        const cssRules = style.innerHTML;
-        const targetTextRules =
-            cssRules.match(/(\w*)::target-text\s*{\s*((.|\n)*?)\s*}/g);
-        if (!targetTextRules) continue;
-
-        const markCss = targetTextRules.join('\n');
-        const newNode = document.createTextNode(markCss.replaceAll(
-            '::target-text', ` .${TEXT_FRAGMENT_CSS_CLASS_NAME}`));
-        style.appendChild(newNode);
-      }
-    };
+export const applyTargetTextStyle = () => {
+  const styles = document.getElementsByTagName('style');
+  if (!styles) return;
+
+  for (const style of styles) {
+    const cssRules = style.innerHTML;
+    const targetTextRules =
+        cssRules.match(/(\w*)::target-text\s*{\s*((.|\n)*?)\s*}/g);
+    if (!targetTextRules) continue;
+
+    const markCss = targetTextRules.join('\n');
+    const newNode = document.createTextNode(markCss.replaceAll(
+        '::target-text', ` .${TEXT_FRAGMENT_CSS_CLASS_NAME}`));
+    style.appendChild(newNode);
+  }
+};
+
+/**
+ * Add color and background-color to the CSS class.
+ *
+ * @param {Object} - background-color and color that will be applied to the
+ *     to the CSS class.
+ */
+export const setDefaultTextFragmentsStyle = ({backgroundColor, color}) => {
```

Comment:
My thinking was that just exporting the tag name would probably be enough for clients that want to restyle this.

I'm not necessarily opposed to having a lib method to handle adding this, but do you have any strong opinions on why it should go here and not elsewhere?